Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM node:8-alpine
+
+# install python
+RUN apk --update add python
+
+# install webpack and http-server
+RUN npm install webpack http-server -g
+
+# Change the working directory
+WORKDIR /tmp
+COPY package.json /tmp/
+
+# Install dependencies
+RUN npm config set registry http://registry.npmjs.org/ && npm install
+
+# Create app directory
+RUN mkdir -p /usr/src/app
+
+# Change the working directory
+WORKDIR /usr/src/app
+
+# Copy content into the working directory
+COPY . /usr/src/app
+
+# copy node_modules into app folder
+RUN cp -a /tmp/node_modules /usr/src/app/
```

Comment:
I don't think this is a good idea because of environment mis-match. Your Mac env may differ from the linux container. better to run npm install, if you need to build multiple times cache will always work for you

Reply:
I had a webpack error that wasn't making sense, when I run npm install from the working dir it can't find some of the webpack owned dependencies so I decided to install in the tmp folder and copy the node_modules folder which seems to be working, I'll try find what causes the error when you install from the working dir.

Reply:
this is a much riskier way of executing rather than getting to the bottom of that issue, this is bound to break at some point you're basically writing a bug haha
look at an example setup [here](https://github.com/andela/micro-authorization-service/blob/develop/Dockerfile.dev)

Reply:
Cool, Thanks.

Reply:
I think that's a private repo, I don't have access.

Reply:
I can help looking into it :-)

Reply:
oh sorry, here's a sample
```
# RUN npm config set registry http://registry.npmjs.org/ && npm install

RUN mkdir -p /usr/src/app
WORKDIR /usr/src/app
```

Reply:
I'll appreciate the help.